packages/vscode-extension/src/debug/common/debugConstants.ts

```diff
@@ -280,3 +283,14 @@ export const RecommendedOperations = Object.freeze({
 export const TeamsFxTaskType = ProductName;
 
 export const DebugNoSessionId = "no-session-id";
+
+export async function isSandboxedEnabled(tokenProvider: M365TokenProvider): Promise<boolean> {
+  const SANDBOX_SENSITIVITY_LABEL = "0fcfd0ff-1cda-407e-bc2b-a350307bd1d5";
```

Comment:
Will sandbox supported for VS TTK? If yes, moving these util to core might be a better choice.

Reply:
Updated, thanks!